lighthouse-viewer/app/src/github-api.js

```diff
@@ -36,6 +36,7 @@ class GithubApi {
       return Promise.reject(new Error('Save already in progress'));
     }
 
+    // @ts-ignore
```

Comment:
what's with all the explanationless ignores? :)

Reply:
I have no idea...all the sudden `tsc -p .` fails because of these. but not in my editor...having weird issues lately :/

Reply:
ok it's b/c the `Strings` import in build-treemap causes TSC to more than it should for `tsc -p .`

Reply:
Have you tried replacing with `window.logger`? That's what's declared on the typedefs.